### DISCUSSION TITLE
handbook/0.1/webnars/20230214_webnar_0002/

### DISCUSSION THREAD

**Discussion by giscus** [transparencia-mg/handbook#44] [Announcements]
# handbook/0.1/webnars/20230214_webnar_0002/

https://transparencia-mg.github.io/handbook/0.1/webnars/20230214_webnar_0002/

**Comment by gabrielbdornas** [comment]
@silvianafaria, @kesiabomfa, @Vilelafm e @Andrelamor, Incluo aqui a documentação de nosso Webnário que ocorreu hoje, 14/02/2023. Esta documentação tem formato de blog, sendo assim, todos podem incluir comentários. Sugiro [esta leitura](https://transparencia-mg.github.io/handbook/0.1/posts/20221226_sistema_de_comentarios/) super rápida para dicas de como escrever/acionar algum colega por aqui.
